Stratis.Bitcoin/BlockPulling/StoreBlockPuller.cs

```diff
@@ -25,14 +25,13 @@ public StoreBlockPuller(ConcurrentChain chain, Connection.IConnectionManager nod
         }
 
         /// <summary>
-        /// Prepares and invokes download task for a single block.
+        /// Prepares and invokes a download task for multiple blocks.
         /// </summary>
-        /// <param name="downloadRequest">Description of a block to download.</param>
-        public void AskBlock(ChainedBlock downloadRequest)
+        public void AskForMultipleBlocks(ChainedBlock[] downloadRequests)
         {
-            this.logger.LogTrace("({0}:'{1}/{2}')", nameof(downloadRequest), downloadRequest.HashBlock, downloadRequest.Height);
+            this.logger.LogTrace("({0}:'{1} to ask.')", nameof(downloadRequests), downloadRequests.Count());
```

Comment:
`this.logger.LogTrace("({0}.{1}:{2})", nameof(downloadRequests), nameof(downloadRequests.Count), downloadRequests.Count());`

Isn't valid, give compiler error ;)